material.css

```diff
@@ -270,3 +270,29 @@ input.Button[type="submit"] {
     border-radius: 18px;
     margin-left: 6px;
 }
+
+.SessionDropdown .Dropdown-toggle .Avatar {
+    margin: 6px 5px -2px -6px;
+    width: 24px;
+    height: 24px;
+    border-radius: 24px;
+    font-size: 12px;
+    line-height: 24px;
+}
+
+@media (min-width: 768px)
+.NotificationsDropdown .Dropdown-toggle {
+    width: 36px;
+    text-align: center;
+}
+
+@media (min-width: 768px)
```

Comment:
Can you remove this line? (289)

Reply:
Do you mean the @ or the whole line? Including the code beneath it?

Reply:
I told the line, meaning you should only delete line 289, not the code under.

Reply:
Ok.